editoast/src/views/infra/auto_fixes/mod.rs

```diff
@@ -182,6 +206,116 @@ fn fix_infra(
     Ok(operations)
 }
 
+fn reduce_operation(op1: Option<Operation>, op2: Option<Operation>) -> Option<Operation> {
+    match (op1, op2) {
+        (Some(Operation::Create(_)), Some(Operation::Create(_)))
+        | (Some(Operation::Create(_)), Some(Operation::Delete(_)))
+        | (Some(Operation::Update(_)), Some(Operation::Create(_)))
+        | (Some(Operation::Delete(_)), Some(Operation::Create(_))) => {
+            error!("cannot reduce these 2 operations, it should never happen");
+            None
+        }
+        (Some(Operation::Create(railjson)), Some(Operation::Update(update))) => {
+            let UpdateOperation {
+                obj_id: id,
+                obj_type: r#type,
+                railjson_patch,
+                ..
+            } = update;
+            debug_assert_eq!(railjson.get_id(), &id);
+            debug_assert_eq!(railjson.get_type(), r#type);
+            serde_json::to_string(&*railjson)
```

Comment:
```suggestion
            serde_json::to_string(&*railjson_object)
```

Reply:
Good point, done in https://github.com/osrd-project/osrd/pull/6012/commits/c2fffa3ec820b08b67c1b8a0d78c4b031048037b.